doc/code/api/pennylane_cirq.SimulatorDevice.rst

```diff
@@ -0,0 +1,93 @@
+SimulatorDevice
+===============
+
+.. currentmodule:: pennylane_cirq
+
+.. autoclass:: SimulatorDevice
+   :show-inheritance:
+
+   .. rubric:: Attributes Summary
+
+   .. autosummary::
+
+      ~SimulatorDevice.author
+      ~SimulatorDevice.circuit_hash
+      ~SimulatorDevice.name
+      ~SimulatorDevice.obs_queue
+      ~SimulatorDevice.observables
+      ~SimulatorDevice.op_queue
+      ~SimulatorDevice.operations
+      ~SimulatorDevice.parameters
+      ~SimulatorDevice.pennylane_requires
+      ~SimulatorDevice.short_name
+      ~SimulatorDevice.shots
+      ~SimulatorDevice.state
+      ~SimulatorDevice.version
+
+   .. rubric:: Methods Summary
+
+   .. autosummary::
+
+      ~SimulatorDevice.active_wires
+      ~SimulatorDevice.apply
+      ~SimulatorDevice.capabilities
+      ~SimulatorDevice.check_validity
+      ~SimulatorDevice.execute
+      ~SimulatorDevice.execution_context
+      ~SimulatorDevice.expval
+      ~SimulatorDevice.generate_samples
+      ~SimulatorDevice.marginal_prob
+      ~SimulatorDevice.post_apply
+      ~SimulatorDevice.post_measure
+      ~SimulatorDevice.pre_apply
+      ~SimulatorDevice.pre_measure
+      ~SimulatorDevice.probability
+      ~SimulatorDevice.reset
+      ~SimulatorDevice.sample
+      ~SimulatorDevice.sample_basis_states
+      ~SimulatorDevice.states_to_binary
+      ~SimulatorDevice.statistics
+      ~SimulatorDevice.supports_observable
+      ~SimulatorDevice.supports_operation
+      ~SimulatorDevice.var
+
+   .. rubric:: Attributes Documentation
+
+   .. autoattribute:: author
+   .. autoattribute:: circuit_hash
+   .. autoattribute:: name
+   .. autoattribute:: obs_queue
+   .. autoattribute:: observables
+   .. autoattribute:: op_queue
+   .. autoattribute:: operations
+   .. autoattribute:: parameters
+   .. autoattribute:: pennylane_requires
+   .. autoattribute:: short_name
+   .. autoattribute:: shots
+   .. autoattribute:: state
+   .. autoattribute:: version
```

Comment:
Here a couple of attributes do not get rendered in the table view as they have no docstrings:
* author
* name
* pennylane_requires
* short_name
* version

We could remove them I guess?

Reply:
@mariaschuld: you'll need to copy the `doc/_templates` folder from PennyLane to here. This includes formatting for the autogenerated documentation page, to avoid issues such as attributes with no docstring!

Reply:
Whoops.